### PR TITLE
Enhance functionality of plan /create

### PR DESCRIPTION
**Pull request by Musfirahe0556596**
Addresses issue #111 . Now users can add whitespaces between commas when they specify the workouts to be added in their plan creation. Additionally, additional test cases have been added.

### CODE REVIEW

File: src/main/java/data/plans/PlanList.java [hunk 1]
```diff
@@ -72,7 +72,7 @@ public Plan createAndAddPlan(String userArgument) throws ArrayIndexOutOfBoundsEx
 
         ArrayList<Workout> workoutsToAddInAPlanList = new ArrayList<Workout>();
         for (int i = 0; i < numberOfWorkoutsInAPlan; i += 1) {
-            int workoutNumberInteger = Integer.parseInt(userWorkoutNumbersString.split(",")[i]);
+            int workoutNumberInteger = Integer.parseInt(userWorkoutNumbersString.split(",")[i].trim());
```
Comment (a1021492980) [inline src/main/java/data/plans/PlanList.java:75]: Effective and accurate detection of bug!

**Review by a1021492980 — APPROVED** [review]
Thank you for fixing bug efficiently.

File: src/main/java/data/plans/PlanList.java (hunk 1 above)
Comment (Musfirahe0556596) [reply src/main/java/data/plans/PlanList.java:75]: props to Alan ^_^!

File: src/test/java/data/plans/PlanListTest.java [hunk 2]
```diff
@@ -64,6 +64,8 @@ void listAllPlan_expectThreePrints() throws InvalidPlanException {
     void createAndAddPlan_validPlansAdded_expectValidPlans() throws InvalidPlanException {
         planList.createAndAddPlan("Plan 1 /workouts 1,5,4,3");
         planList.createAndAddPlan("Plan 2 /workouts 1,5,4,3,1,1,1");
+        planList.createAndAddPlan("Plan 3 /workouts 1, 3, 4");
+        planList.createAndAddPlan("Plan 4 /workouts 1,1, 5,2");
```
Comment (alanlowzies) [inline src/test/java/data/plans/PlanListTest.java:68]: Good additional test cases

**Review by alanlowzies — APPROVED** [review]
Looks good. Thanks for fixing the bug.

**Pull request merged**